### PR TITLE
WM-1591: Include app name in cbas-ui's CBAS URL in COA

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

**Review by kpierre13 — APPROVED** [review]

File: coa-helm/templates/cbas-ui.yaml [hunk 1]
```diff
@@ -21,6 +21,6 @@ spec:
 data:
   {{ .Values.cbasUI.conf_file }}: |-
     {
-      "cbasUrlRoot" : "{{ .Values.relay.path }}/cbas"
+      "cbasUrlRoot" : "{{ .Values.relay.path }}{{ .Values.persistence.leoAppInstanceName }}}}/cbas"
```
Comment (jgainerdewar) [inline coa-helm/templates/cbas-ui.yaml:24]: I think this has double closing braces?

**Review by rtitle — CHANGES_REQUESTED** [review]
As discussed in [slack](https://broadinstitute.slack.com/archives/C02H8KALYMD/p1670507219356029?thread_ts=1670448027.893499&cid=C02H8KALYMD) I think this was a Leo oversight and would be better to fix Leo-side. Here is a ticket: https://broadworkbench.atlassian.net/browse/TOAZ-296

Suggest closing this PR

**Comment by rtitle** [comment]
Leo fix: https://github.com/DataBiosphere/leonardo/pull/3065

**Comment by rtitle** [comment]
The Leo side is merged so this should be fixed for new CBAS apps. Going to close this

**Pull request closed without merging**